test/navigation/NavigatorSpec.scala

```diff
@@ -65,6 +68,26 @@ class NavigatorSpec extends SpecBase {
           .mustBe(routes.BankDetailsController.onPageLoad(NormalMode))
       }
 
+      "go to proofOfAuthority page after after IndirectRepresentative page when the claimant is representative and has selected no" in {
+
+        val answers =
+          emptyUserAnswers
+            .set(ClaimantTypePage, ClaimantType.Representative).success.value
+            .set(IndirectRepresentativePage, false).success.value
+
+        navigator.nextPage(IndirectRepresentativePage, NormalMode, answers)
+          .mustBe(routes.ProofOfAuthorityController.onPageLoad)
+      }
+
+      "go to CheckYourAnswers page after the claimant page once the representative has uploaded their proof of authority" in {
```

Comment:
Can we have a chat about this test? It doesn't look right to me but might be missing something?